school/demo/school_demo.xml

```diff
@@ -482,8 +482,7 @@
 
         <record id="demo_student_news_desc_1" model="student.news">
             <field name="subject">Education Information</field>
-            <field name="date">2013-07-15</field>
+            <field eval="str(int(time.strftime('2017'))) +'-07-13'" name="date"/>
         </record>
-
     </data>
 </odoo>
```

Comment:
Make a new line at the end of file.
Your XML has so many un-necessary whitespaces.